airbyte-to-flow/src/libs/stream.rs

```diff
@@ -261,6 +259,7 @@ mod test {
                 resource_config_json: "{}".to_string(),
                 collection: None,
                 field_config_json_map: BTreeMap::new(),
+                backfill: 7,
```

Comment:
what is this? 😁

Reply:
The `backfill` field was added to the protocol recently (estuary/flow#1276). I had to add it here because I updated the proto-flow dependency.